### PR TITLE
State yamls update

### PR DESCRIPTION
**Pull request by OdeclasV**
Updating WI and CA state yamls

### CODE REVIEW

File: members/WIL000319.yaml [hunk 1]
```diff
@@ -14,25 +14,21 @@ contact_form:
           value: $NAME_FULL
           required: true
         - name: address
-          selector: input#41fda3dd-30b3-4040-f647-06d2a9fc19a8
+          selector: input[id='41fda3dd-30b3-4040-f647-06d2a9fc19a8']
```
Comment (tomdionesotes) [inline members/WIL000319.yaml]: no `#`?

File: members/WIL000339.yaml [hunk 2]
```diff
@@ -3,41 +3,43 @@ contact_form:
...
         - value: Submit
-          selector: ".legis-email-submit-btn"
+          selector: input[type='submit'][value='Submit']
   success:
     headers:
       status: 200
     body:
-      contains: Sign up for
\ No newline at end of file
+      contains: Thank you
```
Comment (tomdionesotes) [inline members/WIL000339.yaml]: does `Thank you` appear anywhere on the form itself? If so there's a chance this causes false positives in terms of detecting successful submissions. Can this success message be made any more unique?

File: members/WIL000448.yaml [hunk 3]
```diff
@@ -3,37 +3,47 @@ contact_form:
...
+        - value: 2
+    - click_on:
+        - selector: li a#ui-id-5
   success:
     headers:
       status: 200
     body:
-      contains: Dear Neighbor
\ No newline at end of file
+      contains: Thank you
```
Comment (tomdionesotes) [inline members/WIL000448.yaml]: Same as above re: unique message

File: members/WIL000347.yaml [hunk 4]
```diff
@@ -21,16 +21,26 @@ contact_form:
...
+        - value: 3
     - find:
-        - selector: p.contourMessageOnSubmit
+        - selector: div .panel-heading
   success:
     headers:
       status: 200
     body:
-      contains: "Thank you"
+      contains: "Feature News"
```
Comment (tomdionesotes) [inline members/WIL000347.yaml]: same as above re: unique message

File: members/WIL000490.yaml [hunk 5]
```diff
@@ -3,37 +3,45 @@ contact_form:
...
         - value: Submit
-          selector: ".legis-email-submit-btn"
+          selector: input[type='submit'][value='Submit']
   success:
     headers:
       status: 200
     body:
-      contains: Hello and welcome
\ No newline at end of file
+      contains: Thank you
```
Comment (tomdionesotes) [inline members/WIL000490.yaml]: Same as above re: unique message

File: members/WIL000347.yaml (hunk 4 above)
Comment (OdeclasV) [reply members/WIL000347.yaml]: This one is kinda tricky. There's no success message and after submitting the form the user is redirected to the home page (https://gyazo.com/bc4b8093bff7edcc47c80752a02dcc21). Of all the text in this page, I thought 'Feature News' was the only constant one that would't change. I also thought about using "Madison Office" (left hand corner), but that one seems to be present in every page you navigate to.  Anything else you see there that I could use?

Comment (tomdionesotes) [reply members/WIL000347.yaml]: ohh man the photo of that kid on the home page.... the most awkward and also the best.

as long as `feature news` will never be present on the page where the form is, that's fine then.

File: members/WIL000490.yaml (hunk 5 above)
Comment (OdeclasV) [reply members/WIL000490.yaml]: Yeah, 'Thank you' seems to be the only text that appears after the form is submitted. It's really annoying, the could be a little bit more creative. 

Here's a screenie of the page after the form has been submitted, [Screenshot from Gyazo](https://gyazo.com/b34731bab00bfa791d22da7a58746199)

And here's a screenie of the with the blank form [Screenshot from Gyazo](https://gyazo.com/f33006b585ecfcb4b36ede5e4d0e175e)

Comment (tomdionesotes) [reply members/WIL000490.yaml]: welp as long as `thank you` never appears it's good. man that is a lazy form.

Comment (OdeclasV) [reply members/WIL000490.yaml]: Indeed. Most of the forms are very similar, but a few do make a greater effort to thank the people that write to them. There are also a few that have been updated more recently (starting last year) but it seems a majority of them just have been left alone for a while.

File: members/WIL000347.yaml (hunk 4 above)
Comment (OdeclasV) [reply members/WIL000347.yaml]: yeah poor kid lol and I didn't notice the hemp agriculture piece, that's really interesting lol

**Pull request merged**